addons/account/tests/test_account_move_entry.py

```diff
@@ -494,6 +494,94 @@ def test_journal_override_sequence_regex(self):
         next.action_post()
         self.assertEqual(next.name, '00000001-G 0002/2017')
 
+
+    def test_journal_override_sequence_draft_move(self):
```

Comment:
This test is not linked to this PR and is already covered by `test_journal_override_sequence_regex`

I think the only thing you are testing here is that the `sequence_override_regex` you are using is valid, not that the flow is working correctly.

Reply:
Yes, you are right. This test won't be part of the final version of this PR.